packages/ts/react-crud/src/locale.ts

```diff
@@ -67,9 +67,7 @@ export class LocaleFormatter {
     if (typeof value === 'object') {
       const { year, month, day } = value;
       const date = new Date();
-      date.setFullYear(year);
-      date.setMonth(month);
-      date.setDate(day);
+      date.setFullYear(year, month, day);
```

Comment:
Could be shortened down to `new Date(year, month, day);` I guess?

```suggestion
      const date = new Date(year, month, day);
```

Reply:
No, it can't. That's what I did at the beginning, but year XX is then converted to 19XX and doesn't pass a test.